src/pages/index.js

```diff
@@ -213,6 +219,31 @@ const IndexPage = () => (
         </div>
       </Section>
     </Slice>
+    <Slice large>
+      <Section title="Contribute">
+        <div className="ml-flexy ml-flexy--items-stretch ml-flexy--gutter">
+          <Tile
+            bottom
+            center
+            title="Contributing"
+            text="Learn how to contribute to Mozaic, report a bug, submit changes or ask help"
+            linkurl="/Contributing/"
+            linktitle="Contribute to Mozaic"
+            link="Discover"
+          ></Tile>
+          <Tile
+            bottom
+            center
+            title="Patterns Status"
+            text="See in real time the status of Mozaic patterns."
```

Comment:
we should say: 
See Mozaic patterns status in real time.

Reply:
@nherwegh as said in a previous feeback upper, i removed the "Pattern status" section.